src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -29,6 +29,10 @@ void LoadARFF(const std::string& filename,
   // First, open the file.
   std::ifstream ifs;
   ifs.open(filename, std::ios::in | std::ios::binary);
+  if (!ifs.is_open())
+  {
+    Log::Fatal << "Cannot open file '" << filename << "'. " << std::endl;
+  }
```

Comment:
Looks like this check is duplicated?

Reply:
Sorry , it was just added to pass the Travis build before @MuLx10 gave his PR.